adminSiteClient/GdocsSettingsForms.tsx

```diff
@@ -32,12 +32,12 @@ const GdocCommonErrors = ({
     )
     return (
         <>
-            {errorsToShowInDrawer.map((error) => (
+            {errorsToShowInDrawer.map((error, index) => (
                 <Alert
+                    key={index}
+                    className="GdocsSettingsForm__alert"
                     message={error.message}
                     type={error.type}
-                    key={error.message}
```

Comment:
We can get multiple errors with the same message, so we can't use the message as the key.